src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -563,7 +563,7 @@ class HeaderBar extends Component {
                       to="/news"
                     />
                   )}
-                  {squadsVisible && (
+                  {/* {squadsVisible && (
```

Comment:
Hi @sunnykarim I think all you have to do is comment out line 467 (like you did): 
const squadsVisible = isWebApp();
...and then set squadsVisible to false on a new line. I don't think you need to comment out any of the other lines.